server/ansible/server-config/roles/pbench-server-activate/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: activate the server configuration
+  shell: PATH=/opt/pbench-server/bin:$PATH pbench-server-activate {{ configdest }}/pbench-server.cfg > /tmp/foo
```

Comment:
If we really don't need the output from `pbench-server-activate` then prehaps `/dev/null` would be better?

Perhaps instead we should consider having `pbench-server-activate` save all this output to `/var/log/pbench-server-activate.log`?

Reply:
Oh - right. I forgot about that: that was debugging. I'll fix.